source/code/source/playfab/PlayFabCurlHttpPlugin.cpp

```diff
@@ -110,6 +112,27 @@ namespace PlayFab
         } // UNLOCK httpRequestMutex
     }
 
+    constexpr char requestIdHeaderKey[] = "X-RequestId:";
+    constexpr char whitespace[] = "\t\n\v\f\r ";
+    size_t HeaderCallback(char* buffer, size_t size, size_t nitems, void* userdata)
+    {
+        size_t headerLen = _countof(requestIdHeaderKey) - 1;
```

Comment:
I would recommend calling this headerKeyLen, since it would be easy to misunderstand its meaning in later lines